launcher/firstLaunch/firstlaunch_moc.cpp

```diff
@@ -627,19 +766,59 @@ bool FirstLaunchView::checkCanInstallTranslation()
 	return checkCanInstallMod(modName);
 }
 
-bool FirstLaunchView::checkCanInstallWog()
+bool FirstLaunchView::checkCanInstallExtras()
 {
-	return checkCanInstallMod("wake-of-gods");
+	return checkCanInstallMod("vcmi-extras");
+}
+
+bool FirstLaunchView::checkCanInstallDemo()
+{
+    if(!checkCanInstallMod("demo-support"))
+        return false;
+
+    QDir userRoot = pathToQString(VCMIDirs::get().userDataPath());
+    QDir dataDir(userRoot.filePath(QStringLiteral("Data")));
+	QDir mapsDir(userRoot.filePath(QStringLiteral("Maps")));
+
+    bool hasDemoMap = false;
+    QStringList mapFiles = mapsDir.entryList(QDir::Files | QDir::Readable);
+    for(const QString &name : mapFiles)
+        if(name.compare(QStringLiteral("h3demo.h3m"), Qt::CaseInsensitive) == 0)
+            hasDemoMap = true;
+	
+    QStringList files = dataDir.entryList(QDir::Files | QDir::Readable);
+    for(const QString &name : files)
+    {
+        if(name.compare(QStringLiteral("H3ab_spr.lod"), Qt::CaseInsensitive) == 0)
+        {
+            QFile lod(dataDir.filePath(name));
+            quint64 fileSize = lod.size();
+			logGlobal->error("H3ab_spr.lod size: %s", static_cast<unsigned long long>(fileSize));
```

Comment:
- `%s` is for strings, most likely you need `%llu`
- unclear why the cast is needed, it's already the same type